lib/tasks/import_address_from_csv.rake

```diff
@@ -25,18 +42,76 @@ namespace :data_import do
       lettings_log.uprn_known = 0
       lettings_log.uprn = nil
       lettings_log.uprn_confirmed = nil
-      lettings_log.address_line1 = row[1]
-      lettings_log.address_line2 = row[2]
-      lettings_log.town_or_city = row[3]
-      lettings_log.postcode_full = row[4]
+      lettings_log.address_line1 = address_line1
+      lettings_log.address_line2 = address_line2
+      lettings_log.town_or_city = town_or_city
+      lettings_log.county = county
+      lettings_log.postcode_full = postcode_full
       lettings_log.postcode_known = lettings_log.postcode_full.present? ? 1 : nil
-      lettings_log.county = nil
       lettings_log.is_la_inferred = nil
       lettings_log.la = nil
```

Comment:
when does la get inferred if postcode is now provided?

Reply:
It would still get inferred as part of before_validation hooks, this is just clearing whatever leftover values might have been here before